### PR TITLE
Fix runtime dependencies (1/n)

### PR DESCRIPTION
**Pull request by MehdiChinoune**

### CODE REVIEW

File: mingw-w64-devcon-git/PKGBUILD [hunk 1]
```diff
@@ -8,17 +8,16 @@ pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
...
 pkgdesc='Search for and manipulate devices (mingw-w64)'
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
 url='https://github.com/Microsoft/Windows-driver-samples/tree/master/setup/devcon'
 license=('custom')
 makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
-             "binutils"
-             "patch")
-depends=()
+             "$([[ ${MINGW_PACKAGE_PREFIX} != *-clang-* ]] || echo binutils)")
```
Comment (mmuetzel) [inline mingw-w64-devcon-git/PKGBUILD:19]: Change to this?
``` diff
-             "$([[ ${MINGW_PACKAGE_PREFIX} != *-clang-* ]] || echo binutils)")
+             $([[ ${MINGW_PACKAGE_PREFIX} != *-clang-* ]] || echo "binutils"))
```

File: mingw-w64-ceres-solver/PKGBUILD [hunk 2]
```diff
@@ -4,15 +4,17 @@ _realname=ceres-solver
...
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=2.0.0
-pkgrel=2
+pkgrel=3
 pkgdesc="A large scale non-linear optimization library (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
 url="http://ceres-solver.org/"
-license=('New BSD license')
+license=('Apache')
```
Comment (lazka) [inline mingw-w64-ceres-solver/PKGBUILD:12]: http://ceres-solver.org/license.html says BSD (??)

Comment (lazka) [reply mingw-w64-ceres-solver/PKGBUILD:12]: ok, there are multiple licenses... well, let's not block this on that then

**Comment by lazka** [comment]
thanks!

**Pull request merged**